packages/nocodb/src/models/BaseUser.ts

```
@@ -175,6 +178,31 @@ export default class BaseUser {
+    ncMeta = Noco.ncMeta,
+  ): Promise<BaseUser> {
+    const baseUser = await ncMeta.metaGet2(
+      context.workspace_id,
+      context.base_id,
+      MetaTable.PROJECT_USERS,
+      {
+        base_id: baseId,
+        fk_user_id: userId,
+      },
+    )
+    return baseUser && this.castType(baseUser);
```

Biome

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)